### ISSUE TITLE
Sidekiq web failed to start with Ruby 2.7 when using active_record_store

### ISSUE THREAD

**Issue by testwisely** [rails/activerecord-session_store#152]
Hi,

I upgraded my app to Ruby 2.7, found the Sidekiq v5.2.7 web failed with the error trace :

```FATAL -- :TypeError (can't quote Rack::Session::SessionId):
activerecord (6.0.2.1) lib/active_record/connection_adapters/abstract/quoting.rb:231:in `_quote'
...
activerecord-session_store (1.1.3) lib/active_record/session_store/session.rb:58:in `find_by_session_id'
```
When I commented out `session_store :active_record_store, ...` in `config/initializers/session_store.rb`, the sidekiq web is fine.  

It might be a Sidekiq issue, post here just in case.

Jason

**Comment by testwisely** [comment]
I think it is Rack issue with Ruby 2.7, OK reverted to 2.0.7.

**Comment by borama** [comment]
I believe this is not related to ruby 2.7 but to rack 2.0.8, and can be probably fixed by #151 .

**Comment by wimpog** [comment]
Has this issue been fixed?
